hack/delete-tekon-and-knative.sh

```diff
@@ -1,133 +1,27 @@
-kubectl delete clusterrolebinding.rbac.authorization.k8s.io eventing-webhook
-kubectl delete clusterrolebinding.rbac.authorization.k8s.io eventing-webhook-resolver
-kubectl delete clusterrolebinding.rbac.authorization.k8s.io eventing-webhook-podspecable-binding
-kubectl delete clusterrole.rbac.authorization.k8s.io service-addressable-resolver
-kubectl delete clusterrole.rbac.authorization.k8s.io serving-addressable-resolver
-kubectl delete clusterrole.rbac.authorization.k8s.io channel-addressable-resolver
-kubectl delete clusterrole.rbac.authorization.k8s.io broker-addressable-resolver
-kubectl delete clusterrole.rbac.authorization.k8s.io messaging-addressable-resolver
-kubectl delete clusterrole.rbac.authorization.k8s.io flows-addressable-resolver
-kubectl delete clusterrole.rbac.authorization.k8s.io eventing-broker-filter
-kubectl delete clusterrole.rbac.authorization.k8s.io eventing-broker-ingress
-kubectl delete clusterrole.rbac.authorization.k8s.io eventing-config-reader
-kubectl delete clusterrole.rbac.authorization.k8s.io channelable-manipulator
-kubectl delete clusterrole.rbac.authorization.k8s.io meta-channelable-manipulator
-kubectl delete clusterrole.rbac.authorization.k8s.io knative-eventing-namespaced-admin
-kubectl delete clusterrole.rbac.authorization.k8s.io knative-messaging-namespaced-admin
-kubectl delete clusterrole.rbac.authorization.k8s.io knative-flows-namespaced-admin
-kubectl delete clusterrole.rbac.authorization.k8s.io knative-sources-namespaced-admin
-kubectl delete clusterrole.rbac.authorization.k8s.io knative-bindings-namespaced-admin
-kubectl delete clusterrole.rbac.authorization.k8s.io knative-eventing-namespaced-edit
-kubectl delete clusterrole.rbac.authorization.k8s.io knative-eventing-namespaced-view
-kubectl delete clusterrole.rbac.authorization.k8s.io knative-eventing-controller
-kubectl delete clusterrole.rbac.authorization.k8s.io knative-eventing-pingsource-mt-adapter
-kubectl delete clusterrole.rbac.authorization.k8s.io builtin-podspecable-binding
-kubectl delete clusterrole.rbac.authorization.k8s.io eventing-sources-source-observer
-kubectl delete clusterrole.rbac.authorization.k8s.io knative-eventing-sources-controller
-kubectl delete clusterrole.rbac.authorization.k8s.io knative-eventing-webhook
-kubectl delete validatingwebhookconfiguration.admissionregistration.k8s.io config.webhook.eventing.knative.dev
-kubectl delete mutatingwebhookconfiguration.admissionregistration.k8s.io webhook.eventing.knative.dev
-kubectl delete validatingwebhookconfiguration.admissionregistration.k8s.io validation.webhook.eventing.knative.dev
-kubectl delete mutatingwebhookconfiguration.admissionregistration.k8s.io sinkbindings.webhook.sources.knative.dev
\ No newline at end of file
+version=v0.23.0
+if [ $1 ]; then
+    version=$1
+fi
+
+# delete Tekton pipeline
+kubectl delete --filename https://openfunction.sh1a.qingstor.com/tekton/pipeline/${version}/release.yaml
```

Comment:
If you take a look at https://github.com/OpenFunction/OpenFunction/blob/main/hack/deploy-tekon-and-knative.sh#L39
You'll find out that we're trying to use google storage for non-China deployment and use qingstor for deployment in China, I think the original method is fine.

Reply:
The original method blocked at `kubectl delete ns xxx`  for minutes of time. I think the removal processing can be improved by using google storage files. The Chinese user blocked by pulling images. The removal processing just use the defination of the yamls. I will convert it.

Reply:
@wanjunlei Pls take a look at your original script to see how to fix the delete ns issue.

Reply:
The reason for blocking at kubectl delete ns xxx is It need delete all resources i the ns before delete ns. The solution is to delete all resources in the ns, and then delete ns.